testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/types/LogResponseActionType.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package africa.absa.testing.scapi.rest.response.action.types
+
+import scala.language.implicitConversions
+
+object LogResponseActionType extends Enumeration {
+  type LogResponseActionType = Value
+
+  val ERROR: LogResponseActionType.Value = Value("error")
+  val WARN: LogResponseActionType.Value = Value("warn")
+  val INFO: LogResponseActionType.Value = Value("info")
+  val DEBUG: LogResponseActionType.Value = Value("debug")
+
+  private val stringToValueMap = values.map(v => v.toString -> v).toMap
+
+  def fromString(s: String): Option[LogResponseActionType] = stringToValueMap.get(s)
+
+  // Implicit conversion from AssertResponseActionType to String
+  implicit def enumValueToString(value: LogResponseActionType): String = value.toString
```

Comment:
```suggestion
  // Implicit conversion from LogResponseActionTypeto String
  implicit def enumValueToString(value: LogResponseActionType): String = value.toString
```

The comment does not match the implementation.
Also, the `enumValueToString` has a name that does not clarify the usecase fully (rather e.g. `logResponseActionType2String`), but `implicit def`s are frowned upon anyway, so maybe it should not be present at all? Or if you must have it, please move it to the test package. e.g. directly to the test `ResponseAssertionsTest` where it seems to be actually used.

Reply:
Fixed in commit 5ec5a73137e30cedef0c0cab5593948d1f27d33c.